README.md

```diff
@@ -2,9 +2,49 @@
 
 > This package includes the Babel preset used by Enact.
 
-## Installation
+## Usage in Enact projects created by Enact CLI
+
+The easiest way to use this configuration is with [Enact CLI](https://github.com/enactjs/cli), which includes it by default. You don’t need to install it separately in Enact projects.
+
+## Usage Outside of Enact projects
+
+If you want to use this Babel preset in a project not built with Enact CLI, you can install it with the following steps.
+
+First, [install Babel](https://babeljs.io/docs/setup/).
+
+Then install babel-preset-enact.
 
 ```sh
-$ npm i --save babel-preset-enact
+npm install babel-preset-enact --save-dev
+```
+
+Then create a file named `.babelrc` with following contents in the root folder of your project:
```

Comment:
Should `.babelrc` be `.babelrc.json` or not?

Reply:
Either is fine, Both are the same.
And CRA also described `.babelrc`.